src/main.rs

```diff
@@ -25,6 +28,9 @@ use std::f64;
 use clap::crate_version;
 
 fn main() {
+  
+  #[cfg(debug_assertions)]
```

Comment:
Why do you need it?

Reply:
[ws-rs](https://github.com/housleyjk/ws-rs) crate uses [env_logger](https://github.com/sebasmagri/env_logger) to dumb out some debug stuff. I was unsure if env_logger should be initialized both in production and development builds.